roles/openshift_logging_elasticsearch/vars/default_images.yml

```diff
@@ -1,3 +1,5 @@
 ---
-__openshift_logging_elasticsearch_proxy_image_prefix: "docker.io/openshift/"
+__openshift_logging_elasticsearch_image_prefix: "{{ openshift_logging_image_prefix | default('docker.io/openshift/origin-') }}"
+__openshift_logging_elasticsearch_image_version: "{{ openshift_logging_image_version | default('latest') }}"
+__openshift_logging_elasticsearch_proxy_image_prefix: "{{ openshift_logging_image_prefix | default('docker.io/openshift/origin-') | regex_replace('$(.*)/origin-', '\\1/') }}"
```

Comment:
starting my regex with `$` instead of `^` ....

Reply:
is it a good idea to use a regexp here? what if a user sets `openshift_logging_image_prefix` to `some-registry/origin-` and is more consistent with his image versioning and really means it? I think it is better not to take our inconsistencies as a precedence